<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingWnd</class>
 <widget class="QWidget" name="CSettingWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1630</width>
    <height>1334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>过滤</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>过滤：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_SelectComboBox">
        <item>
         <property name="text">
          <string>全部定值</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>不分组定值</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>分组定值</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_TypeComboBox">
        <item>
         <property name="text">
          <string>全部类型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>无类型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>控制字</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>电流型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>电压型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>阻抗型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>角度型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>时间型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>计算型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PTCT型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PT型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CT型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CT1次额定值</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CT2次额定值</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>比率系数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>电压滑差</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>字符串型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>定值区号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>系统参数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>长度型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>频率型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>频率滑差型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>出口定值</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>距离保护用角度</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>不受CT二次值影响的电流定值</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>名字包含：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_KeyLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>定值组数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_SettingGroups"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>339</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="m_TableWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>属性</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>二次额定值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>返回系数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最大值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最小值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>缺省值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>位宽</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>精度</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>描述</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>二次值单位</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>一次值单位</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
